src/routeGeneration/routeGenerator.ts

```diff
@@ -3,10 +3,14 @@ import * as path from 'path';
 import * as tsfmt from 'typescript-formatter';
 import { Tsoa } from '../metadataGeneration/tsoa';
 import { fsReadFile, fsWriteFile } from '../utils/fs';
-import { RoutesConfig } from './../config';
+import { RoutesConfig, SwaggerConfig } from './../config';
 import { normalisePath } from './../utils/pathUtils';
 import { TsoaRoute } from './tsoa-route';
 
+interface ISwaggerConfigRelatedToRoutes {
```

Comment:
We're not using the convention to prepend an `I` to interfaces anywhere else, so I wouldn't add it here, just to keep things consistent

Reply:
✔️